src/rust/ide/lib/view/src/graph_editor/builtin/visualization/native/documentation.rs

```diff
@@ -0,0 +1,253 @@
+//! Documentation view visualization generating and presenting Enso Documentation under
+//! the documented node.
+
+use crate::prelude::*;
+
+use crate::graph_editor::component::visualization;
+
+use enso_frp as frp;
+use ensogl::display;
+use ensogl::display::DomSymbol;
+use ensogl::display::scene::Scene;
+use ensogl::system::web;
+use ensogl::system::web::StyleSetter;
+use ast::prelude::FallibleResult;
+
+
+
+// =================
+// === Constants ===
+// =================
+
+pub const DOC_VIEW_WIDTH  : f32 = 300.0;
+pub const DOC_VIEW_MARGIN : f32 = 15.0;
```

Comment:
you are in a `documentation` module. There is no point in repeating `DOC_VIEW_` in constant names.